### ISSUE TITLE
FFMPEG export wrong element-* pattern

### ISSUE THREAD

**Issue by Dragomir-Ivanov** [JonnyBurger/remotion#139]
Greetings,
First, amazing project, just what I needed.
Using it on Linux/Ubuntu with installed ffmpeg from the app center(works just fine).

Having a clip length of 100 frames. Naturally they are all exported to 

```
/tmp/react-motion-XXXX/element-000.jpg
...
/tmp/react-motion-XXXX/element-099.jpg
```

however following command is used to exec `ffmpeg`:
`ffmpeg -r 60 -f image2 -s 1920x1080 -i element-%02d.jpeg -vcodec libx264 -crf 16 -pix_fmt yuv420p outro.mp4`
which fails.

If I change manually `-i element-%02d.jpeg` to `-i element-%03d.jpeg` it works.

I guess this can be fixed with naming the files `element-00.jpg` ... `element-99.jpg`. 
Thanks!

**Comment by yuta0801** [comment]
It would be caused by a bug here.
As the element file index is zero-based, it need to calculate length of frames count - 1 for max index.

https://github.com/JonnyBurger/remotion/blob/0d2aab19ed42bf85db2764e512fe7a1e58fbd5c0/packages/renderer/src/render.ts#L59

**Comment by JonnyBurger** [comment]
Thanks @Dragomir-Ivanov for the excellent bug report and @yuta0801 for tracking it down! Will be released in 1.4.0, probably later today :)
